.eslintrc.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
         "react/forbid-prop-types": "off",
         "react/prefer-stateless-function": "off",
         "react/jsx-curly-brace-presence": ["off", "ignore"],
+        "react/no-array-index-key" : ["off, ignore"],
```

Comment:
I strongly agree with shutting down this warning! Just be careful for sortable or deletable items like table rows in future~ 😛